### PR TITLE
[feature/carousel-component] Add carousel component

### PR DESCRIPTION
**Pull request by bartlomiej-sacha**
Following changes were made:

- carousel component was created
- _slider scss file with custom styles was created
- carousel images were added in assets folder

### CODE REVIEW

File: src/components/Slider/Slider.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react'
+import image1 from '../../assets/guitar-slider1.jpg'
+import image2 from '../../assets/guitar-slider2.jpg'
+import image3 from '../../assets/guitar-slider3.jpg'
+import { Carousel, Button } from 'react-bootstrap';
+
+function Slider() {
+    const [index, setIndex] = useState(0);
+
+    const handleSelect = (selectedIndex, e) => {
```
Comment (Lukortech) [inline src/components/Slider/Slider.js]: "e" seems to be unused here

File: src/components/Slider/Slider.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        <Carousel activeIndex={index} onSelect={handleSelect}>
+            < Carousel.Item interval={1000}  >
+                < img
+                    className="carousel__img"
+                    alt='electric guitar'
+                    src={image1} />
+                < Carousel.Caption >
+                    <div className="carousel__caption-wrapper p-2">
+                        <h3> Electric Guitars </h3 >
+                        <Button className="carousel__caption-btn" variant="primary">CHECK</Button>{' '}
```
Comment (Lukortech) [inline src/components/Slider/Slider.js]: Maybe try using `.text-uppercase` class

File: src/components/Slider/Slider.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+            </Carousel.Item >
+            < Carousel.Item interval={1000} >
+                < img
+                    className="carousel__img"
+                    alt='acoustic guitar'
+                    src={image2} />
+                < Carousel.Caption >
+                    <div className="carousel__caption-wrapper p-2">
+                        <h3> Acoustic Guitars </h3 >
+                        <Button className="carousel__caption-btn" variant="primary">CHECK</Button>{' '}
```
Comment (Lukortech) [inline src/components/Slider/Slider.js]: Ditto

File: src/components/Slider/Slider.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+            </Carousel.Item >
+            < Carousel.Item interval={1000} >
+                < img
+                    className="carousel__img"
+                    alt='guitar pedal'
+                    src={image3} />
+                < Carousel.Caption >
+                    <div className="carousel__caption-wrapper p-2">
+                        <h3> Amplifiers </h3 >
+                        <Button className="carousel__caption-btn" variant="primary">CHECK</Button>{' '}
```
Comment (Lukortech) [inline src/components/Slider/Slider.js]: ditto

File: src/sass/_slider.scss [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  min-width: 100%;
+  object-fit: cover;
+  height: 300px;
+}
+
+.carousel-item {
+  height: 300px;
+
+  &::after {
+    display: block;
```
Comment (Lukortech) [inline src/sass/_slider.scss:11]: `@extend .d-block, .position-relative, .mt-n5, .h-100, .w100;` would be a nice replacement (not sure about the amount of negative margin needed there)

File: src/sass/_slider.scss [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    height: 100%;
+    width: 100%;
+    content: "";
+  }
+}
+
+.carousel {
+  &__caption-wrapper {
+    background-color: rgba(($color-primary), 0.6);
+    border-radius: 5px;
```
Comment (Lukortech) [inline src/sass/_slider.scss]: `@extend .rounded` Could be sufficent
https://getbootstrap.com/docs/4.0/utilities/borders/#border-radius

**Review by Lukortech — APPROVED** [review]
Overall good work, some slight improvements can be done, other than that LGTM

**Comment by bartlomiej-sacha** [comment]
@Lukortech  Thanks for review!

**Pull request merged**